testsuite/features/step_definitions/command_steps.rb

```diff
@@ -616,7 +616,7 @@
   raise 'cobbler sync failed' if out.include? 'Push failed'
 end
 
-When(/^I set the default PXE menu entry to the "([^"]*)" on the "([^"]*)"$/) do |entry, host|
+When(/^I set the default PXE menu entry to the ([^ ]*) on the "([^"]*)"$/) do |entry, host|
```

Comment:
Just a suggestion, but approving anyway.

```suggestion
When(/^I set the default PXE menu entry to the (target profile|local boot) on the "([^"]*)"$/) do |entry, host|
```

Reply:
right, much more legible

Reply:
done